Formula/gcc.rb

```diff
@@ -9,12 +9,11 @@ class Gcc < Formula
     sha256 "ce862b4a4bdc8f36c9240736d23cd625a48af82c2332d2915df0e16e1609a74c"
     version "11.1.0"
```

Comment:
```suggestion
    version "11.2.0"
```

Reply:
@carlocab this is really strange because I used bump-formula-pr. Is it better to close and open again?

Reply:
Just edit this -- `bump-formula-pr` doesn't work for formulae like this one.